### PR TITLE
Adding celestial.hackclub.com

### PR DESCRIPTION
**Pull request by emma-x1**
# Adding celestial.hackclub.com

## Description
Adding page for Celestial, Toronto's Days of Service event!

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -763,6 +763,10 @@ cecclphs:
   - ttl: 3600
     type: TXT
     value: google-site-verification=_c3oOWsKBUxlmaanXeqC1tBTLNZGflwsEMQQiuP-EUM
+celestial:
+  - ttl: 600
+    type: CNAME
+    value: https://emma-x1.github.io/Toronto-DoS-Celestial/
```
Comment (alx-alexpark) [inline hackclub.com.yaml:769]:
```suggestion
    value: emma-x1.github.io
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Hi!!! Just one quick change, could you remove the protocol and the path from the CNAME record as I have suggested? Otherwise, everything looks good! One last thing I would like to ask is if this is a DoS event, shouldn't you have the repo/site on the hackclub github organization?

**Review by alx-alexpark — APPROVED** [review]
yum

File: hackclub.com.yaml (hunk 1 above)
Comment (emma-x1) [reply hackclub.com.yaml]: I just made the change! I just designed the site myself on my personal github and then Zenab told me to add it through this, if I need to make other changes please let me know!

File: hackclub.com.yaml [hunk 2]
```diff
@@ -763,6 +763,10 @@ cecclphs:
   - ttl: 3600
     type: TXT
     value: google-site-verification=_c3oOWsKBUxlmaanXeqC1tBTLNZGflwsEMQQiuP-EUM
+celestial:
+  - ttl: 600
+    type: CNAME
+    value: emma-x1.github.io
```
Comment (alx-alexpark) [inline hackclub.com.yaml:769]:
```suggestion
    value: emma-x1.github.io.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Aaaa forgot that you need the trailing dot

**Review by alx-alexpark — APPROVED** [review]
Ok looks good

**Pull request merged**